tasks/utilities/data_handling/task_fetch_srr_metadata.wdl

```diff
@@ -0,0 +1,56 @@
+version 1.0
+
+task fetch_srr_metadata {
+  input {
+    String sample_accession 
+    String docker = "us-docker.pkg.dev/general-theiagen/biocontainers/fastq-dl:2.0.4--pyhdfd78af_0"
+    Int disk_size = 10
+    Int cpu = 2
+    Int memory = 8
+  }
+  meta {
+    volatile: true
+  }
+
+  command <<< 
+    mkdir -p metadata_output
```

Comment:
do you need to output this information into a directory? I think you could probably do it in the home directory. Just simplifies the code a bit.

Reply:
updated